### PR TITLE
11.0 mig portal partner merge

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**
Depends on:

- [ ] base_partner_merge - #545

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
The Travis will be fixed when the dependency module base_partner_merge is merged.

**Comment by serpentcs-dev1** [comment]
IMHO Can explicitly give the URL in the oca_dependencies.txt file to fix the Travis!

[oca_dependencies.txt](https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt#L15 )

File: portal_partner_merge/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# See LICENSE file for full copyright and licensing details.
+{'name': 'Portal Partner Merge',
+ 'version': '11.0.1.0.0',
+ 'category': 'Hidden',
+ 'author': "Camptocamp,Odoo Community Association (OCA)",
+ 'maintainer': 'Camptocamp',
+ 'license': 'AGPL-3',
+ 'website': 'http://www.camptocamp.com/',
```
Comment (nikul-serpentcs) [inline portal_partner_merge/__manifest__.py:8]: IMHO Please change URL as described in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

**Comment by NL66278** [comment]
Travis failed and no action on comments.

**Pull request closed without merging**